src/auth-guard/auth-guard.ts

```diff
@@ -40,21 +40,23 @@ export class AngularFireAuthGuard implements CanActivate {
     return this.authState.pipe(
       take(1),
       authPipeFactory(next, state),
-      map(can => typeof can == "boolean" ? can : this.router.createUrlTree(<any[]>can))
+      map(can => typeof can === 'boolean' ? can :
+        Array.isArray(can) ? this.router.createUrlTree(can) : this.router.parseUrl(can)
```

Comment:
Can we have a test for this new functionality?

Reply:
The current tests don't seem to test much... I'll see what I can do.